utils.hpp

```diff
@@ -87,15 +87,100 @@ namespace marxan {
             parsed >> value;
         }
 
+        class formatted_string_stream

```

Comment:
there is no built in similar class?

Reply:
I don't see some needed formatting options available in standard stream classes.